introduction.md

```diff
@@ -0,0 +1 @@
+Hello, my name is Ning Wang. This is the second time I'm challenge this dojo.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.